spec/system/candidate_interface/submitting/candidate_reviews_completed_applicatiion_and_updates_qualifications_spec.rb

```diff
@@ -0,0 +1,153 @@
+require 'rails_helper'
+
+RSpec.feature 'Candidate is redirected correctly' do
+  include CandidateHelper
+
+  scenario 'Candidate reviews completed application and updates qualification details section' do
+    given_i_am_signed_in
+    when_i_have_completed_my_application
+    and_i_review_my_application
+    then_i_should_see_all_sections_are_complete
+
+    # GCSE English qualification
+    when_i_click_change_english_gcse_qualification
+    then_i_should_see_the_gcse_type_form
+
+    when_i_click_back
+    then_i_should_be_redirected_to_the_application_review_page
+
+    when_i_update_english_gcse_qualification
+    then_i_should_be_redirected_to_the_application_review_page
+    and_i_should_see_my_updated_gcse_qualification
+
+    # GCSE English grade
+    when_i_click_change_english_gcse_grade
+    then_i_should_see_the_gcse_grade_form
+
+    when_i_click_back
+    then_i_should_be_redirected_to_the_application_review_page
+
+    when_i_update_english_gcse_grade
+    then_i_should_be_redirected_to_the_application_review_page
+    and_i_should_see_my_updated_gcse_grade
+
+    # GCSE English year awarded
+    when_i_click_change_english_gcse_year
+    then_i_should_see_the_gcse_year_form
+
+    when_i_click_back
+    then_i_should_be_redirected_to_the_application_review_page
+
+    when_i_update_english_gcse_year
+    then_i_should_be_redirected_to_the_application_review_page
+    and_i_should_see_my_updated_gcse_year
+  end
+
+  def given_i_am_signed_in
+    create_and_sign_in_candidate
+  end
+
+  def when_i_have_completed_my_application
+    candidate_completes_application_form
+    @current_candidate.current_application.application_references.each do |reference|
```

Comment:
just realised. Do we need to do this?

By default references are received as part of the `candidate_completes_application_form` helper method

![image](https://user-images.githubusercontent.com/42515961/128693273-d6b859b2-b776-4497-a989-14003db8d576.png)

Reply:
![image](https://user-images.githubusercontent.com/42515961/128693344-0c518aac-a22a-42f8-b497-214caf62934b.png)

Reply:
I've removed 👍